### PR TITLE
main/f2fs-tools: upgrade to 1.10.0, clarify license, modernize

### PR DESCRIPTION
**Pull request by prspkt**

### CODE REVIEW

**Comment by nmeum** [comment]
Could you split the upgrade and the modernization into two separate commits? Makes it easier to review this.

**Comment by prspkt** [comment]
@nmeum Yes certainly.

File: main/f2fs-tools/APKBUILD [hunk 1]
```diff
@@ -36,34 +36,23 @@ snapshot() {
...
-	done
-	libtoolize --force && aclocal && autoheader && autoconf \
-		&& automake --add-missing
-}
-
 build() {
-	cd "$_builddir"
+	cd "$builddir"
+	
+	./autogen.sh
```
Comment (nmeum) [inline main/f2fs-tools/APKBUILD]: Could you move the `autogen.sh` invocation to the prepare stage by adding a `prepare()` function the APKBUILD? Also make sure to call default_prepare and make sure you change the directory afterwards.

Comment (prspkt) [reply main/f2fs-tools/APKBUILD]: Sure.

**Comment by prspkt** [comment]
@nmeum The last commit removes snapshotting since upstream provides versioned release packages.

**Pull request closed without merging**